bpf/lib/lb.h

```diff
@@ -77,8 +88,29 @@ struct bpf_elf_map __section_maps LB4_BACKEND_MAP = {
 	.flags          = CONDITIONAL_PREALLOC,
 };
 
+#ifdef ENABLE_SESSION_AFFINITY
+struct bpf_elf_map __section_maps LB4_AFFINITY_MAP = {
+	.type		= BPF_MAP_TYPE_LRU_HASH,
+	.size_key	= sizeof(struct lb4_affinity_key),
+	.size_value	= sizeof(struct lb_affinity_val),
+	.pinning	= PIN_GLOBAL_NS,
+	.max_elem	= CILIUM_LB_MAP_MAX_ENTRIES,
+	.flags		= 0,
+};
+#endif
+
 #endif /* ENABLE_IPV4 */
 
+#ifdef ENABLE_SESSION_AFFINITY
+struct bpf_elf_map __section_maps LB_AFFINITY_MATCH_MAP = {
+	.type		= BPF_MAP_TYPE_HASH,
+	.size_key	= sizeof(struct lb_affinity_match),
+	.size_value	= sizeof(__u8),
```

Comment:
What does __u8 stand for? Maybe a struct should be defined here as well?

Reply:
This is a dummy val, as this map is used as a set. Can I create a map which val size is 0?

Reply:
Yeah indeed after reading the later patches. Map size 0 is not possible. But comment in the code would help that it's used as a set instead.